src/05-derivatives/03-imaging.md

```diff
@@ -189,25 +209,27 @@ pipeline/
             sub-001_space-orig_dseg.json
 ```
 
-A segmentation could be a binary mask that functions as a discrete `label` for a
-single structure. In this case, the label key must be used to specify the
-corresponding structure. For example:
+A segmentation can be used to generate a binary mask that functions as a
+discrete "label" for a single structure.
+In this case, the mask suffix MUST be used, and the `label` entity SHOULD be used
+to specify the masked structure (see [Anatomical labels](#anatomical-labels), and
+the `Atlas` metadata SHOULD be defined.
+For example:
 
 ```Text
 pipeline/
     sub-001/
         anat/
-            sub-001_space-orig_label-GM_dseg.nii.gz
+            sub-001_space-orig_label-GM_mask.nii.gz
 ```
 
-See [Anatomical Labels](#anatomical-labels) for reserved key values for `label`.
-
 ### Probabilistic Segmentations
 
-Probabilistic segmentations of brain tissue represent a single tissue class with
-values ranging from 0 to 1 in individual 3D volumes or across multiple frames.
-Similarly to a discrete, binary segmentation, the label key can be used to
-specify the corresponding structure.
+Probabilistic segmentations of brain tissue represent a single anatomical
+structure with values ranging from 0 to 1 in individual 3D volumes or across
```

Comment:
```suggestion
structure with values in the range `[0.0,1.0]` in individual 3D volumes or across
```

(trying to indicate the the indicated range is inclusive of the bounding values)